packages/lesmis/src/routes/PageRoutes.jsx

```diff
@@ -19,8 +19,8 @@ import { VerifyEmailView } from '../views/VerifyEmailView';
 import { ABOUT_PAGE, FQS_PAGE, CONTACT_PAGE } from '../constants';
 import { ExportView, PDF_DOWNLOAD_VIEW } from '../views/ExportView';
 import { getAdminApiUrl } from '../utils/getAdminApiUrl';
+import AdminPanel from './AdminPanelApp';
 
-const AdminPanel = lazy(() => import('./AdminPanelApp'));
```

Comment:
Changing this from a lazy import to a standard es module import will increase the bundle size slightly on the main js bundle but will fix the circular dependency issue so is a trade off worth doing in my opinion. Also we have dramatically reduced the main bundle size already by using chunking in the js build so overall it's still better than it was.